### PR TITLE
Use tls_skip_verify in vault-ssh-helper

### PR DESCRIPTION
**Pull request by vishalnayak**

### CODE REVIEW

File: api/ssh_agent.go [hunk 1]
```diff
@@ -80,7 +81,7 @@ func (c *SSHHelperConfig) NewClient() (*Client, error) {
 	clientConfig.Address = c.VaultAddr
 
 	// Check if certificates are provided via config file.
-	if c.CACert != "" || c.CAPath != "" {
+	if c.CACert != "" || c.CAPath != "" || c.TLSSkipVerify {
```
Comment (jefferai) [inline api/ssh_agent.go]: This check  for `c.TLSSkipVerify` doesn't actually affect any logic below.

Comment (vishalnayak) [inline api/ssh_agent.go]: It does invoke `c.SetTLSParameters`

Comment (jefferai) [inline api/ssh_agent.go]: Ah, OK. I see what it's doing there. :+1:

**Comment by jefferai** [comment]
:ship:

**Pull request merged**